src/iframe-helper.js

```diff
@@ -346,12 +350,13 @@ export function postMessage(iframe, type, object, targetOrigin, opt_is3P) {
  * @param {!Object} object Message payload.
  * @param {boolean=} opt_is3P set to true if the iframe is 3p.
  */
-export function postMessageToWindows(iframe, targets, type, object, opt_is3P) {
+export function postMessageToWindows(
+    iframe, sentinel, targets, type, object, opt_is3P) {
```

Comment:
while I think you are right, getting rid of that parameter might be better suited to another PR. I suspect getting rid of it in this one will lead to many additional changes